core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,39 @@
+@charset "UTF-8";
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number (unitless)} $width [1]
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The type of grid for this column. By default, the global
+///   `$neat-default-grid` will be used.
```

Comment:
I think it is good to start referring to 'the grid' as a thing that you create and that there are multiples of. preceding with : `The grid to be used to generate the column`